### PR TITLE
began refining the OVN procedure

### PR DESCRIPTION
**Pull request by klgill**
https://issues.redhat.com/browse/RHOSPDOC-1926

@booxter or @slawqo Can you please review my edits to the procedure and answer the 2 questions I put in this PR? Thanks!

### CODE REVIEW

File: docs_user/modules/proc_migrating-ovn-data.adoc [hunk 1]
```diff
@@ -102,31 +98,31 @@ spec:
...
+$ oc exec ovn-copy-data -- bash -c "ovsdb-client backup --ca-cert=/etc/pki/tls/misc/ca.crt --private-key=/etc/pki/tls/misc/tls.key --certificate=/etc/pki/tls/misc/tls.crt ssl:$SOURCE_OVSDB_IP:6642 > /backup/ovs-sb.db"
 ----
 
-. Start control plane OVN database services prior to import, while keeping `northd` and `ovn-controller` disabled.
+. Start the control plane OVN database services prior to import, with `northd` and `ovn-controller` disabled:
 +
 [source,yaml]
 ----
-oc patch openstackcontrolplane openstack --type=merge --patch '
+$ oc patch openstackcontrolplane openstack-galera-network-isolation --type=merge --patch '
```
Comment (klgill) [inline docs_user/modules/proc_migrating-ovn-data.adoc:125]: Step 5: Should the number of replicas be 1? 
https://github.com/openstack-k8s-operators/architecture/pull/305/files

File: docs_user/modules/proc_migrating-ovn-data.adoc [hunk 2]
```diff
@@ -212,25 +210,27 @@ spec:
 '
 ----
 
-. If you'd like to run OVN gateway services on OCP nodes, also enable the control plane `ovn-controller` service:
+. Optional: To run OVN gateway services on {OpenShiftShort} nodes, enable the control plane `ovn-controller` service:
```
Comment (klgill) [inline docs_user/modules/proc_migrating-ovn-data.adoc:213]: Step 11: Is this step actually optional? Or is it conditional, i.e., it only applies to customers who have OVN gateway services on OCP nodes?

File: docs_user/modules/proc_migrating-ovn-data.adoc [hunk 3]
```diff
@@ -2,18 +2,15 @@
...
- ** The `OpenStackControlPlane` resource must be already created at this point.
- ** `NetworkAttachmentDefinition` CRDs for the original cluster are already defined. Specifically, `internalapi` network is defined.
- ** The original {networking_service} and OVN `northd` are not running.
- ** There must be network routability between control plane services and the adopted cluster.
-* Define the following shell variables. The values that are used are examples. Replace these example values with values that are correct for your environment:
+* The `OpenStackControlPlane` resource is created.
+* `NetworkAttachmentDefinition` custom resources (CRs) for the original cluster are defined. Specifically, the `internalapi` network is defined.
+* Ensure that the original {networking_first_ref} and OVN `northd` are not running.
+* Ensure that there is network routability between the control plane services and the adopted cluster.
+* Define the following shell variables. Replace the example values with values that are correct for your environment:
```
Comment (slawqo) [inline docs_user/modules/proc_migrating-ovn-data.adoc:13]: There is one more prerequisite which we never mentioned I think and this is that You have to be on the ML2/OVN already. It may be kind of obvious but in OSP 17.1 ML2/OVS is still supported backend so it has to be migrated to ML2/OVN before adoption to RHOSO will be started. I created Jira https://issues.redhat.com/browse/OSPRH-8725 but didn't had time to do it yet.

File: docs_user/modules/proc_migrating-ovn-data.adoc [hunk 4]
```diff
@@ -24,23 +21,22 @@ ifeval::["{build}" == "downstream"]
 STORAGE_CLASS=local-storage
 OVSDB_IMAGE=registry.redhat.io/rhosp-dev-preview/openstack-ovn-base-rhel9:18.0
 endif::[]
-SOURCE_OVSDB_IP=172.17.1.49
+SOURCE_OVSDB_IP=172.17.0.100
```
Comment (slawqo) [inline docs_user/modules/proc_migrating-ovn-data.adoc:24]: why this change of IP? It isn't any problem for me, I am just curious :)

File: docs_user/modules/proc_migrating-ovn-data.adoc (hunk 1 above)
Comment (slawqo) [reply docs_user/modules/proc_migrating-ovn-data.adoc:125]: I think it should be set like that already, this step is just to update it from "enabled: false" to "enabled:true"

File: docs_user/modules/proc_migrating-ovn-data.adoc (hunk 2 above)
Comment (slawqo) [reply docs_user/modules/proc_migrating-ovn-data.adoc:213]: the latter, it only applies to the users who have OVN gateways on OCP nodes

File: docs_user/modules/proc_migrating-ovn-data.adoc (hunk 4 above)
Comment (klgill) [reply docs_user/modules/proc_migrating-ovn-data.adoc:24]: @slawqo I'm sorry, I should've explained. This was a minor suggestion from Jiri to align with the rest of the examples in the guide. This is his comment from hackfest: https://docs.google.com/document/d/17GzJf8uTfpe7TJrEpf9NomZ98LcnNvUs2V3j9jgDw2w/edit?disco=AAABOWBvSeI

File: docs_user/modules/proc_migrating-ovn-data.adoc (hunk 3 above)
Comment (klgill) [reply docs_user/modules/proc_migrating-ovn-data.adoc:13]: Ok, for now I added the following prerequisite:
The cloud is migrated to the Modular Layer 2 plug-in with Open Virtual Networking (ML2/OVN) mechanism driver.

**Comment by rheslop** [comment]
LGTM
